PowerValidatedSolutions.psm1

```diff
@@ -10522,6 +10522,233 @@ Function Undo-NsxtNodeProfileSyslogExporter {
+		.SYNOPSIS
+        Returns the vRealize Log Insight logging configuration (CFAPI) on vRealize Automation.
+
+        .DESCRIPTION
+        The Get-vRAvRLIConfig cmdlet returns the vRealize Log Insight logging
+        configuration for vRealize Automation.
+        The cmdlet connects to SDDC Manager using the -server, -user, and -password values and
+        connects to the first vRealize Automation appliance using the -rootPass value.
+        - Validates that network connectivity and authentication is possible to SDDC Manager
+        - Validates that network connectivity and authentication is possible to Management Domain vCenter Server
+        - Validates that network connectivity is possible to the first vRealize Automation appliance
+        - Returns the vRealize Log Insight configuration in vRealize Automation
+
+        .EXAMPLE
+        Get-vRAvRLIConfig -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -rootPass VMw@re1!
+        This example returns the vRealize Log Insight logging configuration on vRealize Automation.
+    #>
+
+    Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$server,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$user,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$pass,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$rootPass
+    )
+
+    Try {
+        if (Test-VCFConnection -server $server) {
+            if (Test-VCFAuthentication -server $server -user $user -pass $pass) {
+                if (($vcfVcenterDetails = Get-vCenterServerDetail -server $server -user $user -pass $pass -domainType MANAGEMENT)) {
+                    if (Test-VsphereConnection -server $($vcfVcenterDetails.fqdn)) {
+                      if (Test-VsphereAuthentication -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass) {
+                            if ($vraDetails = Get-vRAServerDetail -fqdn $server -username $user -password $pass -ErrorAction Stop) {
+                                if (Test-vRAConnection -server $vRADetails.node1IpAddress) {   
+                                    $vmName = $vraDetails.fqdn | Select-Object -First 1
+                                    $vmName = $vmName.Split(".")[0]
+                                    if ((Get-VM -Name $vmName -WarningAction SilentlyContinue -ErrorAction SilentlyContinue )) {
+                                        $scriptCommand = "vracli vrli"
+                                        $output = Invoke-VMScript -VM $vmName -ScriptText $scriptCommand -GuestUser root -GuestPassword $rootPass -Server $vcfVcenterDetails.fqdn
+                                        if (($output.ScriptOutput).Contains('No vRLI integration configured')) {
+                                            Write-Output "vRealize Automation integration with vRealize Log Insight not set: SKIPPED"
```

Comment:
Confused by the Write-Output. You have SKIPPED but all others functions only use this with a Write-Warning.
Suggest we remove the SKIPPED part as nothing is really skipped your simply getting the current state.

Maybe `vRealize Automation integration with vRealize Log Insight status 'Not Configured'`